src/sign-in-xplat/README.md

```diff
@@ -0,0 +1,113 @@
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                         | Notes                                                                                                       |
+|:------------------------------:|:------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-dotnet-xplat-maui`                              | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **My organization only**                                          | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | `Mobile and desktop applications`                                 | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **Redirect URIs**              | `https://login.microsoftonline.com/common/oauth2/nativeclient`    | Required value for this sample. <br/> You can change that later in your own implementation.                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box or select in the Azure portal.
+
+<details>
+   <summary>:computer: Alternative: Register the application using az-cli</summary>
+
+1. Register a new Azure AD App with a reply url
+
+   ```bash
+   AZURE_AD_APP_CLIENT_ID_XPLAT=$(az ad app create --display-name "active-directory-dotnet-xplat-maui" --reply-urls "https://login.microsoftonline.com/common/oauth2/nativeclient" --native-app true --query appId -o tsv)
+   ```
+
+</details>
+
+### 2. Configure the web app
+
+1. Open the `XPlat.csrpoj` under the the `sign-in-xplat` folder in your code editor.
+1. Open the `MainPage.xaml.cs` file and modify the following code:
+
+    ```csharp
+    private const string ClientId = "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]";
+    private const string Tenant = "[Copy the Tenant Id from the Azure portal]";
+    ```
+
+## Run the application
+
+### 1. Run the .NET MAUI App
+
+1. Press `F5` and ensure you have selected `Windows Machine`
+
+### 2. Signin into the app
+
+1. Once the app is running you can sign-in with your user credentials.
+
+### 3. Clean up
+
+1. Delete the Azure AD app
+
+   ```bash
+   az ad app delete --id $AZURE_AD_APP_CLIENT_ID_XPLAT
+   ```
```

Comment:
~~~suggestion
~~~

This only works if they've done the optional Azure CLI method of app registration. I recommend *not* including a *Clean Up* section.

Reply:
that is absolutely @mmacy 

removed | addressed [cc93e](from https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/cc93ea488cfeaa78dc84722d4bf82405603b271c)